src/base/detoken.lua

```diff
@@ -49,23 +49,33 @@
 			-- convert the token into a function to execute
 			local func, err = loadstring("return " .. token)
 			if not func then
-				return nil, err
+				return nil, "load error: " .. err
 			end
 
 			-- give the function access to the project objects
 			setfenv(func, e)
 
 			-- run it and get the result
-			local result = func() or ""
+			local success, result = pcall(func)
```

Comment:
This change in behaviour; `result` is no longer coerced to `""` if `func()` returns `nil`, seem to cause problems...